core/templates/components/state-editor/state-editor.directive.ts

```diff
@@ -130,6 +140,12 @@ angular.module('oppia').directive('stateEditor', [
             $scope.interactionIdIsSet = false;
             $scope.servicesInitialized = false;
             $scope.stateName = StateEditorService.getActiveStateName();
+            ctrl.explorationId = ContextService.getExplorationId();
+            $http.get('/createhandler/data/' + ctrl.explorationId)
```

Comment:
Use UrlInterpolationService to construct the URL.

Reply:
Done.